WordPress/Classes/ViewRelated/Post/Prepublishing/PrepublishingSocialAccountsViewController.swift

```diff
@@ -168,17 +191,26 @@ extension PrepublishingSocialAccountsViewController {
 private extension PrepublishingSocialAccountsViewController {
 
     var enabledCount: Int {
-        connections.filter { $0.isOn }.count
+        connections
+            .map { connectionChanges[$0.keyringID] ?? $0.isOn }
+            .filter { $0 }
+            .count
```

Comment:
Super minor and up to you if you want to change it. I believe the `map` can be removed here.
 ```suggestion
        connections
            .filter { connectionChanges[$0.keyringID] ?? $0.isOn }
            .count
```

Reply:
Good idea! Applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/21188/commits/e36337e1e9a02f6865290bd31180be3574922a0f.